Samples/StyleTransfer/AppViewModel.cs

```diff
@@ -0,0 +1,636 @@
+using GalaSoft.MvvmLight.Threading;
+using System.Threading;
+using StyleTransferEffectComponent;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Devices.Enumeration;
+
+namespace StyleTransfer
+{
+    class AppViewModel : INotifyPropertyChanged
+    {
+        // Media capture properties
+        public MediaCapture _mediaCapture;
+        private bool _isPreviewing;
+        private DisplayRequest _displayrequest = new DisplayRequest();
+        DeviceInformationCollection _devices;
+        private readonly object _processLock = new object();
+        StyleTransferEffectComponent.StyleTransferEffectNotifier _notifier;
+
+        // Style transfer effect properties
+        private LearningModel _learningModel = null;
+        private LearningModelDeviceKind _inferenceDeviceSelected = LearningModelDeviceKind.Default;
+        private LearningModelSession _session;
+        private LearningModelBinding _binding;
+        private string _inputImageDescription;
+        private string _outputImageDescription;
+        // Activatable Class ID of the video effect. 
+        private const string StyleTransferEffectId = "StyleTransferEffectComponent.StyleTransferEffect";
+
+        // Image style transfer properties
+        uint _inWidth, _inHeight, _outWidth, _outHeight;
+        private const string DefaultImageFileName = "Input.jpg";
+
+        // User notifiaction properties
+        private bool _succeed;
+        private readonly SolidColorBrush _successColor = new SolidColorBrush(Windows.UI.Colors.Green);
+        private readonly SolidColorBrush _failColor = new SolidColorBrush(Windows.UI.Colors.Red);
+        public AppViewModel()
+        {
+            _appModel = new AppModel();
+            _appModel.PropertyChanged += _appModel_PropertyChanged;
+
+            SaveCommand = new RelayCommand(SaveOutput);
+            SetMediaSourceCommand = new RelayCommand<InputMediaType>((x) => _appModel.InputMedia = x);
+            SetModelSourceCommand = new RelayCommand(async () => await SetModelSource());
+            ChangeLiveStreamCommand = new RelayCommand(async () => await ChangeLiveStream());
+
+            _inputSoftwareBitmapSource = new SoftwareBitmapSource();
+            _outputSoftwareBitmapSource = new SoftwareBitmapSource();
+            _saveEnabled = true;
+
+            _notifier = new StyleTransferEffectComponent.StyleTransferEffectNotifier();
+            _notifier.FrameRateUpdated += async (_, e) =>
+            {
+                await DispatcherHelper.RunAsync(() =>
+                {
+                    RenderFPS = e;
+                });
+
+            };
+            UseGpu = false;
+            _isPreviewing = false;
+            NumThreads = 5;
+            NotifyUser(true);
+        }
+
+        private AppModel _appModel;
+        public AppModel AppModel
+        {
+            get
+            {
+                return _appModel;
+            }
+        }
+        private int _numThreads;

```

Comment:
nit: I would move all properties next to member declarations, and separate each property with a blank newline for readability.